packages/wonder-blocks-modal/components/one-pane-dialog.stories.js

```diff
@@ -0,0 +1,203 @@
+        width: 874,
+        height: 551,
+        position: "absolute",
+        top: 40,
+        left: -140,
+    },
+
+    below: {
+        background: "url(/modal-below.png)",
+        width: 868,
+        height: 521,
+        position: "absolute",
+        top: -100,
+        left: -300,
+    },
+});
+
+export const kitchenSink = () => {
+    const modal = (
+        <OnePaneDialog
+            title="Single-line title"
+            subtitle="Subtitle that provides additional context to the title"
+            testId="one-pane-dialog-above"
+            content={
+                <View>
+                    <Body>
+                        {
+                            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."
+                        }
+                    </Body>
+                    <br />
+                    <Body>
+                        {
+                            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."
+                        }
+                    </Body>
+                    <br />
+                    <Body>
+                        {
+                            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est."
+                        }
+                    </Body>
+                </View>
+            }
+            footer={<Button>Button (no-op)</Button>}
+            closeButtonVisible={true}
+            above={<View style={styles.above} />}
+            below={<View style={styles.below} />}
+        />
+    );
+
+    return (
+        <ModalLauncher
+            modal={modal}
+            testId="modal-launcher-default-example"
+            opened={true}
+            onClose={() => alert("This would close the modal.")}
+        />
+    );
+};
+
+export const withOpener = () => {
+    type MyModalProps = {|
```

Comment:
This is a major selling point. Having the ability of adding types to our code examples!